### PR TITLE
Cleaning up some graphics files

### PR DESCRIPTION
**Pull request by JoelPasvolsky**
Most graphics are under ``_images`` but a handful were under ``_static``. I ``grep``ped through all and removed a few I don't see used anywhere and think no longer are. My local build is not whining.

### CODE REVIEW

File: docs/_images/theme_overrides.css
Comment (randomir) [inline docs/_images/theme_overrides.css:1]: This file is still used in `docs/conf.py`.

File: docs/_static/inspector_AND_broken_chain.png
Comment (randomir) [inline docs/_static/inspector_AND_broken_chain.png:1]: Used in `docs/examples/and.rst`.

**Review by randomir — COMMENTED** [review]
Looks like a couple of the deleted files is still in use -- or am I missing something?

File: docs/_static/inspector_AND_broken_chain.png
Comment (JoelPasvolsky) [reply docs/_static/inspector_AND_broken_chain.png:1]: Please check again: I think that [docs/examples/and.rst](https://raw.githubusercontent.com/dwavesystems/dwave-ocean-sdk/master/docs/examples/and.rst) post-[PR 253](https://github.com/dwavesystems/dwave-ocean-sdk/pull/253) uses ``.. figure:: ../_images/inspector_AND_broken_chain.png`` (from ``images``)

File: docs/_images/theme_overrides.css
Comment (JoelPasvolsky) [reply docs/_images/theme_overrides.css:1]: The file under ``_images`` directory and not [this one under docs/_static](https://github.com/dwavesystems/dwave-ocean-sdk/blob/master/docs/_static/theme_overrides.css) directory?

Comment (randomir) [reply docs/_images/theme_overrides.css:1]: Oh, I missed that one, you're right.

**Review by randomir — APPROVED** [review]

**Review by arcondello — APPROVED** [review]

**Pull request merged**